libraries/botbuilder/src/consoleAdapter.ts

```diff
@@ -196,15 +206,17 @@ export class ConsoleAdapter extends BotAdapter {
      * Logs text to the console.
      * @param line Text to print.
      */
-    protected print(line: string) {
+    protected print(line: string): void {
+        // tslint:disable-next-line:no-console
```

Comment:
If you use `// tslint:disable:no-console`, it will actually disable the rule throughout the file so you don't need to add it for [every line](https://palantir.github.io/tslint/usage/rule-flags/).